source/features/sticky-sidebar.css

```diff
@@ -45,6 +45,10 @@ Exclusively use simple sums and use `0px` instead of `0`
 	z-index: 1;
 }
 
+#discussion_bucket #partial-discussion-sidebar {
```

Comment:
Makes sure that it doesn't apply when the feature is disabled
```suggestion
#discussion_bucket.rgh-sticky-sidebar-container #partial-discussion-sidebar {
```

Reply:
It should be `#discussion_bucket #partial-discussion-sidebar.rgh-sticky-sidebar-container`

![image](https://github.com/user-attachments/assets/b0b61d00-3656-4f59-b658-a9a8bce9d314)